librivox.yaml

```diff
@@ -29,18 +29,3 @@
     - { 'role': 'maintenance', 'tags': 'maintenance' }
   handlers:
     - import_tasks: handlers.yaml
-
```

Comment:
I appreciate your input, but what I have removed is actually the part that is supposed to run only for the localdev server. That's because I couldn't make Ansible to only run the `localdev` part. The original docs say this needs to run to install LibriVox on localdev:
```
ansible deploy.yml -i hosts/localdev/hosts
```
but the `deploy.yml` file was removed in `2689deba` and I assume was merged into `librivox.yaml`. With the contents of the `hosts/localdev/hosts` file being something like this:
```
[server]
192.168.100.23
```
When I run
```
ansible librivox.yaml -i hosts/localdev/hosts
```
I get the Ansible help with the following error:
```
Error! No argument passed to command module.
```

I don't have experience with Ansible and the Ansible docs on module seems to be something totally different from what I expect. Changing the command to
```
ansible librivox.yaml -m localdev -i hosts/localdev/hosts
```
gives me this output
```
[WARNING]: Could not match supplied host pattern, ignoring: librivox.yaml

[WARNING]: No hosts matched, nothing to do
```

If I change the `hosts/localdev/hosts` file to this:
```
[localdev]
192.168.100.23
```
and I run the command again, I get the same error (or warning).


From looking at an earlier [pull request](https://github.com/LibriVox/librivox-ansible/pull/12), I see that using `ansible-playbook` might work. So when I run
```
ansible-playbook librivox.yaml -i hosts/localdev/hosts
```
I get this error:
```
ERROR! Attempting to decrypt but no vault secret found.
```
which I suppose means that Ansible is running the tasks for production because when I pull the `localdev` part out of librivox.yaml and put it in a new file and then run the command again, I can get Ansible to run the playbook and LibriVox is installed.

What might be misleading is that I pulled the `localdev` part out of `librivox.yaml` into a file called `localdev.yaml` but since there was already a file with the same name, I renamed that file to `localdev-vars.yaml`.

My goal here is to make the `ansible` command only run the `localdev` part of the librivox.yaml file. How do I do that?